config/resources/lpdc.json

```diff
@@ -68,6 +68,11 @@
           "predicate": "pera:language",
           "cardinality": "many"
         },
+        "review-status":{
+          "target": "concept",
+          "predicate": "ext:reviewStatus",
+          "cardinality": "one"
+        },
```

Comment:
I think this should best be moved to the "public-services" resource since it only applies to that one, right?